src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -68,9 +68,8 @@ class ReportActionItemMessageEdit extends React.Component {
      */
     updateDraft(newDraft) {
         this.textInput.setNativeProps({text: newDraft});
-        const trimmedNewDraft = newDraft.trim();
```

Comment:
Why are we just getting rid of the trim? I assume it was there for a reason right?

Reply:
we used selection in TextInputFocusable props so when we update the message then selection(cursor position) also update and if we use space at the end of the message then trim will remove that space so selection(cursor position) goes beyond the message length so it'll create issue

Reply:
So would a better solution then be to change the `cursor position`? Something like this maybe?

```
updatedCursorPosition = cursorPosition > message.length ? message.length : cursorPosition;
```

Reply:
I think it makes sense to trim drafts and get rid of useless whitespace in messages

Reply:
@stitesExpensify any update about the above-mentioned solution?